src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,73 @@
+package duke.task;
+
+import duke.exception.DukeException;
+import duke.exception.UnknownTaskException;
+import duke.main.Storage;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList implements Serializable {
+    List<Task> taskList = new ArrayList<>();
```

Comment:
maybe rename 'taskList' as 'tasks' for it to be plural form.

Reply:
Resolving this as per my comment on the first instance of _"Collection Naming Conventions"_ being pointed out! Thanks for being thorough, nonetheless!